packages/legend-application-studio/src/index.ts

```diff
@@ -93,6 +93,7 @@ export { getElementIcon } from './components/shared/ElementIconUtils.js';
  */
 
 export * from './stores/DSL_Generation_LegendStudioApplicationPlugin_Extension.js';
+export * from './stores/STO_ProjectOverview_LegendStudioApplicationPlugin_Extension.js';
```

Comment:
`getExtraTestRunnerTabsClassifiers` --> `getExtraTestRunnerTabClassifiers`
`getExtraTestRunnerTabsEditorRenderers` --> `getExtraTestRunnerTabEditorRenderers`

Update documentation

The name of this file seems really odd. Also, because `TestRunner` is part of core studio, we can add these to `LegendStudioApplicationPlugin`